gips/0080-optimistic-ebo.md

```diff
@@ -0,0 +1,237 @@
+- Any staked actor can also challenge the proposed update. If a dispute arises, the initial proposal is discarded, and a new one can be submitted.
+- If a dispute occurs, the resolution mechanism is triggered, and the winner of the dispute will receive the counterparty’s bond as a reward.
+- If no disputes are raised within the designated time window, or if the dispute is resolved in favor of the proposer, the update is considered valid.
+
+Operators can update the block number of one chain at a time, allowing for ongoing disputes and resolutions for each updated chain. This improves the system's scalability.
+
+The [Detailed Specification](https://www.notion.so/DRAFT_I-GIP-0XYZ-Optimistic-EBO-74a0b9df7008464e9dc1215c46920213?pvs=21) section will provide further details on how contracts and off-chain agents interact.
+
+### Timestamp Comparison
+
+Implementing a slight delay in timestamp comparison allows for safer EBO operations, significantly reducing risks related to RPC liveness and fork choice (finality) in exchange for a slight they delay in reward distribution:
+
+- **Check the timestamp from the first block of the new epoch in the protocol chain (Arbitrum One).**
+- **Check the block number for each indexed chain at that timestamp.**
+
+The following methods can be used to prove that a specific block number corresponds to a claimed timestamp:
+
+- Check the timestamps of two consecutive blocks $t_N$ and $t_{N+1}$. If the timestamp value from the protocol chain $t_0$ lives in the range $[t_N,t_{N+1}]$, then $N$ is the first block corresponding to the new epoch.
+- Check a single block timestamp $t_N$ and an average block measure defined per chain $\Delta_i$. If the timestamp value from the protocol chain $t_0$ lives in the range $[t_N,t_N+\Delta_i]$, then $N$ is the first block corresponding to the new epoch.
+
+### Escalating bond
+
+To reduce the number of calls to The Graph's Arbitrator, we will implement an escalating bond system:
+
+- Anyone can dispute while the dispute window is open and if staked. To dispute, you need to post a bond.
+    - The resolution system is triggered once the participants have completed the maximum number of rounds.
+    - If the disputant “overtakes” the Min Bond, the answer is considered incorrect until the proposer responds. The proposer must overtake the disputant's Bond to escalate the answer further. This process can be repeated up to N times.
+- After each dispute, the original update gets deleted and can be answered again, even before resolution. This option makes the process quicker for the requestor. This answer can be disputed using the same method as described above.
+    - An honest proposer will be rewarded more for the won resolution than the original update.
+    - Disputers are incentivized to post the new answer, as they have already computed the query.
+- The payment from the requester will end up going to the proposer with the correct answer
```

Comment:
Term 'requester' is used just twice (with different spellings) and not defined. Is the requester the dispute raiser (might be simpler to say disputer)?

I believe that this should not be the entire payment.

Reply:
The requester is the one who requests data from Prophet and also sets the rewards for proposers to submit responses. The disputer monitors the proposals and challenges them if they are invalid.